documentation/modules/ref-operator-cluster.adoc

```diff
@@ -121,6 +118,30 @@ The DNS domain name is added to the Kafka broker certificates used for hostname
 +
 If you are using a different DNS domain name suffix in your cluster, change the `KUBERNETES_SERVICE_DNS_DOMAIN` environment variable from the default to the one you are using in order to establish a connection with the Kafka brokers.
 
+[discrete]
+== Logging configuration
+
+A `ConfigMap` containing logging configuration is created when installing the Cluster Operator.
+This `ConfigMap` is described in the file `install/cluster-operator/050-ConfigMap-strimzi-cluster-operator-logging.yaml`.
```

Comment:
I would avoid using "logging" in the name tbh. We don't know if at some point we will need to add more into this `ConfigMap` and having "logging" in the name could be misleading. Wdyt?